scripts/Michael-Ashi.py

```diff
@@ -1,6 +1,6 @@
 name = "Michael Ashi"
 myId = "HNG-01089"
-language = "python"
+language = "Python"
 email = "ashimichaelashi@gmail.com"
 
 print("Hello World, this is " + name + " with HNGi7 ID " + myId + " using " + language + " for stage 2 task " + email + "")
```

Comment:
email should come after ID.  
the format is **"Hello World, this is [fullname] with HNGi7 ID [ID] and email [email] using [language] for stage 2 task"**